server/belga/publish/belga_newsml_1_2.py

```diff
@@ -418,10 +418,11 @@ def _format_picture(self, newscomponent_1_level, picture):
         :param Element newscomponent_1_level: NewsComponent of 1st level
         :param dict picture: picture item
         """
-        guid = picture.get('guid') or picture['_id']
-        if guid in self._seen_pictures:
+        # picture with same language not exported multiple times in output
+        image_id = (picture.get('guid') or picture['_id']) + picture.get('language', 'en')
```

Comment:
does that language play any role? the language wouldn't change imo unless the guid/id changes

Reply:
Yes, Before what happening here is, this check is only for `guid/id` so after translation of the main item it is creating only one picture item, 
but now we have a check that `guid/id + language` so after translation we have two picture items in the output XML

For example: suppose we have one main item with one picture association(current language NL) then while translating the main item(suppose we converted from NL to FR language) and then publish the main item-
**Before:** 

> In output XML we have two main item(one for NL and another for FR) and only one picture item(for NL only)(according to SDBELGA-514)

**Now:** 

> In output XML we have two main items (one for NL and another for FR) and two picture items (for NL and FR also)(according to SDBELGA-597)

**I made changes as per the comment mentioned by Gideon and that is** 

> Here is the explanation from the original ticket in which we excluded the image from the payload **#SDBELGA-514**
> You have closed ticket **#SDBELGA-597** because the request conflicts with this ticket, where we had asked you not to export the same image ID twice.
> 
> It appears that this request was incorrect. Instead of deduplicating based on the image ID only, it should be based on the combination (image ID + language), which is the unique combination to our business. Therefore I reopen this ticket and ask you to alter the deduplication rule accordingly.

Reply:
that all looks about right, but the picture language will be the same, when you translate an article we don't translate it's media, and if we did then that would create new picture items with different id, so this wouldn't be a problem in the first place. the language there should be the text item language.

Reply:
seems like the language is inherited from the text item, can you add a comment regarding this?